### DISCUSSION TITLE
Can non-insider achieve this function

### DISCUSSION THREAD

**Discussion by AIGC2024** [squidfunk/mkdocs-material#7089] [Q&A] [answered]
![image](https://github.com/squidfunk/mkdocs-material/assets/152939692/e4f8936e-8569-4105-a2a6-c3eba5673b7e)

I'm sorry , I was a beginner of material . As described in the screenshot, what is this function called and how do I implement it?

**Comment by shenweiyan** [comment, marked as answer]
It's called [Instant Previews](https://github.com/squidfunk/mkdocs-material/issues/6704), a new feature in the insiders version.

**Reply by AIGC2024** [reply]
Cool, thanks.
